src/spLoginForm.tpl.html

```diff
@@ -51,11 +55,11 @@
           <a href="/forgot" class="pull-right">Forgot Password</a>
         </div>
       </div>
-      <div class="form-group" ng-show="socialLoginProviders.length">
+      <div class="form-group" ng-show="viewModel.accountStores.length">
```

Comment:
I only have the GitHub account store activated. If there's no supported account stores, then the element will be shown without any provider buttons.

See: https://www.dropbox.com/s/yftfkq5n6nobcka/Screenshot%202016-03-01%2016.33.09.png?dl=0